src/components/Cards.js

```diff
@@ -180,25 +144,47 @@ function Cards(props) {
       </CardWrapper>
     )
   }
+
   return (
     <CardWrapper>
-      {NotStacked.map(card => (
-        <CardItem>
-          <CardLink to={card.to}>
+      {questions.map((card, index) => (
```

Comment:
This means you can just reuse `Card.jsx` here as well.

Reply:
Thanks :-)